internal/core/software_renderer.rs

```diff
@@ -60,6 +62,95 @@ pub enum RepaintBufferType {
     SwappedBuffers,
 }
 
+/// Specify that the window should be drawn rotated
+///
+/// Argument to be passed in [`SoftwareRenderer::set_window_rotation`]
+#[non_exhaustive]
+#[derive(Default, Copy, Clone, Eq, PartialEq, Debug)]
+pub enum WindowRotation {
+    /// No rotation
+    #[default]
+    NoRotation,
+    /// Rotate 90° to the left
+    Rotate90,
+    /// 180° rotation (upside-down)
+    Rotate180,
+    /// Rotate 90° to the right
+    Rotate270,
+}
+
+impl WindowRotation {
+    fn transposed(self) -> bool {
```

Comment:
I suggest to call it is_transposed() to disambiguate it from a function that returns the rotation transposed.